index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+
+  <nav id="navbar">
+  <header>User's Manual</header>
+ 
+    
+    <ul>
+      <li><a class="nav-link" href="#Setting_up_your_oven"> Setting up your oven </a></li>
+      <li><a class="nav-link" href="#Clock_Setting"> Clock Setting </a></li>
+      <li><a class="nav-link" href="#Operation"> Operation </a></li>
+      <li><a class="nav-link" href="#Maintenance"> Maintenance </a></li>
+      <li><a class="nav-link" href="#Warranty_information"> Warranty information </a></li>
+       </nav>
+
+<head>
```

Comment:
In your project, the `<head>` tag starts in line 16 and ends in line 83. The `<head>` section shouldn't be bigger than `<body>,` where your website's content will actually become visible. Kindly check this w3schools article in order to understand better the function of the `<head>` tag: [https://www.w3schools.com/tags/tag_head.asp](url). This will also solve many semantic problems in your project.